build/root/WORKSPACE

```diff
@@ -66,3 +50,10 @@ docker_pull(
     registry = "gcr.io",
     repository = "google-containers/debian-iptables-amd64",
 )
+
+docker_pull(
+    name = "official_busybox",
+    registry = "index.docker.io",
+    repository = "library/busybox",
+    tag = "latest",  # TODO: more hermetic?
```

Comment:
I can, but then I'm not sure when we'd bump it.

For `debian-iptables-amd64`, we know when we're updating it, so it's easy to update the bazel dependency at the same time.
There isn't a clear indication for `busybox`, since we don't control it.
@timstclair thoughts?